bindings/tls.c

```diff
@@ -78,6 +78,7 @@ uintptr_t _solo5_tls_data_offset(uintptr_t tls)
 #if defined(__x86_64__) || defined(__powerpc64__)
     data = tls;
 #elif defined(__aarch64__)
+    data = tls;
     data = data + sizeof(struct tcb);
```

Comment:
shouldn't these two lines be `data = tls + sizeof(struct tcb);`? The `data = data..` looks fishy.

Reply:
I will push on `master` directly with your change indeed :+1:.